test/driver.cpp

```diff
@@ -52,50 +52,51 @@ void processCmdLine(char* arguments[], const size_t size)
    string_view fileOpenMode;
    string outputFilename;
 
-   constexpr string_view name_header{ "-h" }, name_header_text{ "-ht" },
-      name_summary{ "-s" }, namePRM{ "-p" }, name_file{ "-f" },
-      name_file_overwrite{ "-fo" }, name_file_append{ "-fa" },
-      name_threshold{ "-t" };
+   constexpr string_view option_name_header{ "-h" }, option_name_header_text{ "-ht" },
+      option_name_summary{ "-s" }, option_name_prm{ "-p" }, option_name_file{ "-f" },
+      option_name_file_overwrite{ "-fo" }, option_name_file_append{ "-fa" },
+      option_name_threshold{ "-t" };
 
-   constexpr string_view value_yes{ "yes" };
-   constexpr string_view value_detail{ "detail" },
-      value_indicate{ "indicate" }, value_none{ "none" }, value_auto{ "auto" };
+   constexpr string_view option_value_yes{ "yes" };
+   constexpr string_view option_value_detail{ "detail" },
+      option_value_indicate{ "indicate" }, option_value_none{ "none" },
+      option_value_auto{ "auto" };
 
    //begin parsing cmd-line arguments
    for (size_t i = 1; i < size; i += 2)
    {
-      string_view name(arguments[i]), value(arguments[i + 1]);
+      string_view option_name(arguments[i]), value(arguments[i + 1]);
```

Comment:
I suggest reverting the identifier `option_name` back to just `name` for both simplicity and to be consistent with the name `value`. These two variables have pretty short scope and thus are quite visible.